site/styles/_nav.scss

```diff
@@ -847,3 +847,15 @@ s72-dropdown,
     display: none;
   }
 }
+
+.nav-banner {
+  text-align: center;
+  position: relative;
+  z-index: 10;
+  background-color: $primary;
+  padding: 5px;
+  .nav-banner-message {
+      display: contents;
```

Comment:
Why do we need this? [For Beirut I got rid of the inner element and just had the text in the `div` with reduced styles](https://github.com/shift72/beirut-dc/commit/a4dfddc9f351bce063638ff04737a952c0f95065). I want to know if I'm missing something?

Reply:
I think had to make few tweaks like adjusting the padding, position, etc to make it fit for the new navbar. I guess Beirut is still on the old one.